### ISSUE TITLE
Sprites 

### ISSUE THREAD

**Issue by daggerdagger307** [daggerdagger307/brendanwilson.ca#16]
Hello, @thomasjbradley 

I'm playing with this glitch effect on my buckets and I'm running into a few issues. 
1. The sprite file is huge, I'm going to compress them further and take out the drop shadow from the file but will I be able to get the under 130k?
2. I'm using px to define my sprite "window" and because of this it's no longer responsive which is why when the page grows it leaves it's bounding box. Is there any way to make it responsive and stay with in the box?

Sorry I took such a 90deg turn with my site but I'm starting to look for internships and the way I had it didn't reflect the kind of job I'm going for. I'm really very excited about it now and think it could be a portfolio piece in itself. 

Thanks 

-Brendan

**Comment by daggerdagger307** [comment]
So I can't for the life of me get my png's under 130k the best I could do was 546kb but even with just the one gem my fans turn on on my computer. I'll put it back to the buildings before class on Thursday. At least that version is done.

**Comment by daggerdagger307** [comment]
@thomasjbradley 

To switch back to an older version of my site I have downloaded the old files from the repository I'd like to use. Do I simply cut all the files from my folder and paste in the new ones? 

Thanks 

-Brendan

**Comment by thomasjbradley** [comment]
Yeah, you can just download the old versions and overwrite the code with the ones you downloaded.
